src/frontend/admin/public/mockServiceWorker.js

```diff
@@ -2,7 +2,7 @@
 /* tslint:disable */
 
```

Comment:
Commit body suggestion: 

`With Playwright, there is a command to generate tests graphically. I added this command to the package.json to avoid retyping it every time.`